projects/packages/my-jetpack/_inc/components/stats-section/cards.jsx

```diff
@@ -11,19 +11,23 @@ import styles from './style.module.scss';
  * @param {object} props                - Component props.
  * @param {object} props.counts         - Counts object for the current period.
  * @param {object} props.previousCounts - Counts object for the previous period.
+ * @param {number} props.headingLevel   - Heading level between 1 and 6.
  *
  * @return {object} StatsCards React component.
  */
-const StatsCards = ( { counts, previousCounts } ) => {
+const StatsCards = ( { counts, previousCounts, headingLevel } ) => {
 	return (
 		<div className={ styles[ 'section-stats-highlights' ] }>
-			<h3 className={ styles[ 'section-title' ] }>
-				<span>{ __( '7-day highlights', 'jetpack-my-jetpack' ) }</span>
-
-				<small className={ styles[ 'section-description' ] }>
-					{ __( 'Compared to previous period', 'jetpack-my-jetpack' ) }
-				</small>
-			</h3>
+			{ React.createElement(
+				`h${ headingLevel >= 1 && headingLevel <= 6 ? headingLevel : 3 }`,
+				{ className: styles[ 'section-title' ] },
+				<>
+					<span>{ __( '7-day highlights', 'jetpack-my-jetpack' ) }</span>
+					<small className={ styles[ 'section-description' ] }>
+						{ __( 'Compared to previous period', 'jetpack-my-jetpack' ) }
+					</small>
+				</>
+			) }
```

Comment:
Another option here would be to use JSX directly to create the component: 
```
const HeadingTag = `h${headingLevel >= 1 && headingLevel <= 6 ? headingLevel : 3}`;
```

... and then to replace the `React.createElement` code:
```
			<HeadingTag className={styles['section-title']}>
				<span>{__('7-day highlights', 'jetpack-my-jetpack')}</span>
				<small className={styles['section-description']}>
					{__('Compared to previous period', 'jetpack-my-jetpack')}
				</small>
			</HeadingTag>
```
Primarily, the suggestion is for better readability and maintainability. What do you think?